test_benchmarks.py

```
@@ -41,3 +44,84 @@
+        cache = RequestCache(cache_dir=temp_dir, max_age=3600)
+        url = "http://example.com/benchmark/set"
+        content = "<html><body>Test content for benchmarking cache set</body></html>" * 10
+        
+        def cache_set():
+            cache.set(url, content)
+            
+        benchmark(cache_set)
+
+
+@pytest.mark.benchmark(group="caching")
+def test_benchmark_cache_get(benchmark):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_benchmark_cache_get`

Add return type annotation: `None`

(ANN201)